sdk/src/test/scala/com/horizen/account/receipt/EthereumReceiptTest.scala

```diff
@@ -187,7 +187,7 @@ class EthereumReceiptTest
       //println("cumGas =" + cumGas.toString())
       val logsBloom = new Array[Byte](256)
       val logs = new ListBuffer[EvmLog]
-      receipts(i) = new EthereumConsensusDataReceipt(txType, status, cumGas, logs, logsBloom)
+      receipts(i) = new EthereumConsensusDataReceipt(txType, status, cumGas, logs,new LogsBloom(logsBloom))
```

Comment:
We can remove `val logsBloom = new Array[Byte](256)`
Then use another constructor `receipts(i) = new EthereumConsensusDataReceipt(txType, status, cumGas, logs)` so to be sure that bloom filter is consistent to `logs`